about.html

```diff
@@ -0,0 +1,176 @@
+      <a href="index.html"><img src="images/header-logo.png" alt="header logo" class="header-logo" /></a>
+
+      <!-- #### navbar-section #### -->
+      <nav>
+        <div class="desktop-nav">
+          <ul class="desktop-menu-items">
+            <li>
+            <a class="desktop-menu-item" href="index.html">Home</a>
+            </li>
+             <li>
+            <a class="desktop-menu-item" href="#">Program</a>
+            </li>
+             <li>
+            <a class="desktop-menu-item" href="#">Tutors</a>
+            </li>
+             <li>
+            <a class="desktop-menu-item" href="#partners">Sponser</a>
+             </li>
+             <li>
+            <a class="desktop-menu-item" href="#contact">Contact</a>
+            </li>
+            <li>
+            <a class="desktop-menu-item1" href="#">CC Campaign</a>
+            </li>
+          </ul>
+        </div>
+        <div class="mobile-nav">
+     <button type="button" class="btn-open" ><img alt="hamburger icon" src="images/hamburger-icon.png" class="hamburger-icon" /></button>  
+        <div class="mobile-menu-container">
+              <button type="button" class="btn-close">X</button>
+          <ul class="mobile-menu-items">
+            <li>
+            <a class="mobile-menu-item" href="index.html">Home</a>
+            </li>
+             <li>
+            <a class="mobile-menu-item" href="#">Program</a>
+            </li>
+             <li>
+            <a class="mobile-menu-item" href="#">Tutors</a>
+            </li>
+             <li>
+            <a class="mobile-menu-item" href="#partners">Sponser</a>
+            </li>
+            <li>
+            <a class="mobile-menu-item" href="#contact">Contact</a>
+            </li>
+          </ul>
+        </div>
+        </div>
+      </nav>
+      </div>
+    </header>
+
+    <!-- #### main-section #### -->
+    <main>
+
+      <!-- #### hero-section #### -->
+      <section class="hero-section">
+        <h2 class="headline-heading1">"Hello! technology geeks"</h2>
+        <h2 class="headline-heading2">
+          BECOME A PROFESSIONAL REMOTE WEB DEVELOPER
+        </h2>
+        <p class="headline-details">
+          We teach the world Web development remotly, joining our community will let you have many oppurtunity for your future. We are producing decent web developers around the world. We are hoping to expend our origination with many indivisuals
+        </p>
+        <div class="headline-img-container">
+          <div>
+        <h2 class="headline-heading3">NEXT COHORT STARTING IN 15 DAYS</h2>
+        <p class="headline-details2">Book your enrolment now</p>
+        </div>
+        <img class="hero-img" alt="hero image" src="images/hero-bg.png">
+        </div>
+        <p class="headline-details">Please contact us per email for any further quastions about TheCodeChaser bootcamp</p>
+        <p class="email">contact@thecodechaser.com<i class="fas fa-chevron-right"></i></p>
+      </section>
```

Comment:
- [ ] The `hero` section (first section of the home page) should fill the whole screen view on `desktop` sized screens, the `font-size` is a bit smaller compared to the design. Kindly update the `hero` section to match the design.
- [ ] _[OPTIONAL]_ Pleas add a background image to the hero section that has a similar color scheme to the one found in the design.
- [ ] Content inside the `hero` section should be aligned a bit to the left on `desktop` screens.
Reference Design
![image](https://user-images.githubusercontent.com/33767951/142095151-85a9ec27-2b49-4fc1-a9a4-ca6374a2bf0c.png)
Your Styling => '786px'
![image](https://user-images.githubusercontent.com/33767951/142400427-61140846-6b35-43a9-baa3-845a2b8a99f7.png)